composer.json

```diff
@@ -29,6 +29,7 @@
         "symfony/routing": "^2.3 || ^3.0"
     },
     "suggest": {
+        "doctrine/orm": "^2.0",
```

Comment:
This is wrong. Please add a text instead of the suggested version.

Reply:
ping @maarekj